README.md

```diff
@@ -14,13 +14,13 @@ SPDX v3.0 release, leading to breaking changes in the API.
 Please refer to the [migration guide](https://github.com/spdx/tools-python/wiki/How-to-migrate-from-0.7-to-0.8)
 to update your existing code.
 
-We encourage new users to work with v0.8.0rc1 directly as the older v0.7 release is in maintenance mode.
-
 The main features of v0.8 are:
-- experimental support for the upcoming SPDX v3 specification (note, however, that support is neither complete nor 
-  stable at this point, as the spec is still evolving)
 - full validation of SPDX documents against the v2.2 and v2.3 specification
-- support for SPDX's RDF format with all v2.3 features.
+- support for SPDX's RDF format with all v2.3 features
+- experimental support for the upcoming SPDX v3 specification. Note, however, that support is neither complete nor 
+  stable at this point, as the spec is still evolving. The code from the "spdx3" package is not yet in use
```

Comment:
I find this a bit confusing. Perhaps something like this?

SPDX3 support is contained in a separate submodule `spdx3` and its use is optional. We do not yet recommend using it in production code.